dotcom-rendering/src/components/HeaderTopBar.importable.tsx

```diff
@@ -21,6 +22,7 @@ interface HeaderTopBarProps {
 	idApiUrl: string;
 	headerTopBarSearchCapiSwitch: boolean;
 	hasPageSkin?: boolean;
+	pageId?: string;
```

Comment:
Can this ever be `undefined`?

Reply:
I could default `pageId` to be an empty string in the caller component, though I don't know if this is better or worse?